src/CMake/native.cmake

```diff
@@ -25,10 +25,11 @@ ELSE(OPENCL_FOUND)
   MESSAGE(FATAL_ERROR "Looking for OPENCL - not found")
 ENDIF(OPENCL_FOUND)
 
+# -- Git --
 find_package(Git)
 
 IF(GIT_FOUND)
-  message("git found: ${GIT_EXECUTABLE}")
+  message("-- Looking for GIT - found at ${GIT_EXECUTABLE}")
```

Comment:
Should this be "message(STATUS "Looking for GIT - found at ${GIT_EXECUTABLE}")" ?

Reply:
Update this pull request with the requested changes.